plow_gui/Cargo.toml

```diff
@@ -9,9 +9,9 @@ repository = "https://github.com/field33/plow/"
 edition = "2021"
 
 [dependencies]
-plow_ontology = { path = "../plow_ontology", version = "0.1.0" }
-plow_package_management = { path = "../plow_package_management", version = "0.1.0" }
-plow_linter = { path = "../plow_linter", version = "0.1.0" }
+plow_ontology = "0.1"
+plow_package_management = "0.1"
+plow_linter = "0.1"
```

Comment:
Are they on purpose out of sync with `0.2`?

Reply:
Yep, I omit maintaining the gui currently to manage time.